Makefile

```diff
@@ -7,7 +7,7 @@ OUT_DIR=_output
 BIN?=prom-label-proxy
 VERSION?=$(shell cat VERSION)
 PKGS=$(shell go list ./... | grep -v /vendor/)
-DOCKER_REPO?=quay.io/openshift/prom-label-proxy
+DOCKER_REPO?=quay.io/coreos/prom-label-proxy
```

Comment:
silly question: shouldn't these all be `openshift`?

Reply:
silly answer: I don't have permission to create repos under that, this seemed like the path of least resistance.